test/sign.js

```diff
@@ -76,6 +76,16 @@ describe('Offline Signing Tool', function() {
     txHex.should.equal('{"compression":"none","packed_trx":"d3ca235dbed650bc3a64000000000100a6823403ea3055000000572d3ccdcd01507551997772734c00000000a8ed323221507551997772734c9052d3d42ec9b071a08601000000000004454f53000000000000","signatures":["SIG_K1_JzpSv5pkpzuzg5FqYUAVsrEtPooVCDx5Ls3qkpyuSpsZozj6cMCxgz2jYuuqBoLBTRPet62QFoaVVJ6Rrh1YDp64yRMBBy","SIG_K1_K6YVr5Mhgw4JC4vd4r9v2SgLTRTK8JhnuffK8BCMXPBdht77jyoWZHJQzZbwwEvCY4LhTv6Fnnuey1ibnZniJWzrF6y5sZ"]}');
   });
 
+  it('cosigns a trx transaction', function() {
+    const recoveryRequest = JSON.parse(fs.readFileSync('./test/transactions/ttrx.json', { encoding: 'utf8' }));
+    const key = 'xprv9s21ZrQH143K2SGfLqMk9eaSbix4XUqXg2wqXkATpfnQsyvaXBTnEqi71aLSq1rL3qJh32FRrA2VnrfMMEmbNS5xnRCiNSHKdAVR6Ep5Ptx';
```

Comment:
how did you create this wallet with an xprv? It seems like TRX keycards store an encrypted version of a Tron Private Key (looks like: ```64673EC824572D3D1123E3E77704D01F3F76F75344D45A2FEE016C8E3F478D10```) rather than xprv format

Reply:
update: @argjv and @twtaylorbitgo say they are going to update bitgojs so keys are stored as xprv

Reply:
Yeah and it doesn't matter for the sake of signing here.. Account lib only needs the Pk bytes